pkg/apis/policy/fuzzer/fuzzer.go

```diff
@@ -40,6 +40,12 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 			}
 			psp.RunAsUser.Rule = runAsUserRules[c.Rand.Intn(len(runAsUserRules))]
 
+			runAsGroupRules := []policy.RunAsGroupStrategy{
```

Comment:
note: [MayRunAs](https://github.com/kubernetes/kubernetes/pull/65135) should be added here once that PR merges

Reply:
It merged. I think you should update this PR to include that strategy.

Reply:
fixed